resource-managers/kubernetes/integration-tests/pom.xml

```diff
@@ -29,7 +29,7 @@
     <download-maven-plugin.version>1.3.0</download-maven-plugin.version>
     <exec-maven-plugin.version>1.4.0</exec-maven-plugin.version>
     <extraScalaTestArgs></extraScalaTestArgs>
-    <kubernetes-client.version>4.1.2</kubernetes-client.version>
+    <kubernetes-client.version>4.1.0</kubernetes-client.version>
```

Comment:
I presume this just needs to be reverted

Reply:
This was an attempt at seeing what's going on in jenkins: see SPARK-26997.